app/messaging/inbound/process-generation.js

```diff
@@ -6,23 +6,30 @@ const { updateStatus } = require('../../services/documents')
 const status = require('../../constants/document-status')
 
 const processGenerationRequest = async (message, receiver) => {
+  let documentId
+
   try {
     const body = validateGenerationMessage(message.body)
+    documentId = body.document_id
```

Comment:
document_id comes from the message body which is returned by the validate function. Validate throws an exception so we need to use let so it can be set inside the try block.